content/desktop/install/msi/install-and-configure.md

```diff
@@ -0,0 +1,197 @@
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /quiet /norestart
+```
+
+#### Installing non-interactively with admin settings
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /quiet /norestart ADMINSETTINGS="{"configurationFileVersion":2,"enhancedContainerIsolation":{"value":true,"locked":false}}" ALLOWEDORG="docker.com"
+```
+
+#### Installing with the passive display option
+
+You can use the `/passive` display option instead of `/quiet` when you want to perform a non-interactive installation but show a progress dialog.
+
+In passive mode the installer doesn't display any prompts or error messages to the user and the installation cannot be cancelled.
+
+For example:
+
+```powershell
+msiexec /i "DockerDesktop.msi" /L*V ".\msi.log" /passive /norestart
+```
+
+> **Tip**
+>
+> Some useful tips to remember when creating a value that expects a JSON string as it’s value:
+> 
+> - The property expects a JSON formatted string
+> - The string should be wrapped in double quotes
+> - The string should not contain any whitespace
+> - Property names are expected to be in double quotes
+{ .tip }
+
+### Common uninstall commands
+
+When uninstalling Docker Desktop, you need to use the same `.msi` file that was originally used to install the application.
+
+If you no longer have the original `.msi` file, you need to use the product code associated with the installation. To find the product code, run:
+
+```powershell
+Get-WmiObject Win32_Product | Select-Object IdentifyingNumber, Name | Where-Object {$_.Name -eq "Docker Desktop"}
+```
+
+It should return output similar to the following:
+
+```text
+IdentifyingNumber                      Name
+-----------------                      ----
+{10FC87E2-9145-4D7D-B493-2E99E8D8E103} Docker Desktop
+```
+> **Note**
+>
+> This command can take some time to return, depending on the number of installed applications.
+
+`IdentifyingNumber` is the applications product code and can be used to uninstall Docker Desktop. For example:
+
+```powershell
+msiexec /x {10FC87E2-9145-4D7D-B493-2E99E8D8E103} /L*V ".\msi.log" /quiet
+```
+
+
+
+#### Uninstalling interactively with verbose logging
+
+```powershell
+msiexec /x "DockerDesktop.msi" /L*V ".\msi.log"
+```
+
+#### Uninstalling interactively without verbose logging
+
+```powershell
+msiexec /x "DockerDesktop.msi"
+```
+
+#### Uninstalling non-interactively with verbose logging
+
+```powershell
+msiexec /x "DockerDesktop.msi" /L*V ".\msi.log" /quiet
+```
+
+#### Uninstalling non-interactively without verbose logging
+
+```powershell
+msiexec /x "DockerDesktop.msi" /quiet
+```
+
+### Configuration options 
+
+> **Important**
+>
+> In addition to the following custom properties, the Docker Desktop MSI installer also supports the standard [Windows Installer command line options](https://learn.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options).
+{ .important }
+
+| Property | Description | Default |
```

Comment:
@aevesdocker Please could you add a new property to the table?

`DISABLEANALYTICS` | When set to `1`, analytics collection will be disabled for the MSI.